src/components/Footer/Footer.styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export const Wrapper = styled.footer`
+  display: flex;
+  background-color: #8f4456;
```

Comment:
We should add this color to the theme palette on the theme.ts file

```suggestion
  display: flex;
  background-color: ${({ theme }) => theme.colors.maroon};
```